apps/console/src/features/applications/components/wizard/minimal-application-create-wizard.tsx

```diff
@@ -183,6 +169,18 @@ export const MinimalAppCreateWizard: FunctionComponent<MinimalApplicationCreateW
             return;
         }
 
+        /**
+         * Remove the default callbackURLs from the form values if theres any user defined values
+         * (callbackURLs) in the form. We do this to prevent appending the default `callbackURL`
+         * to the model.
+         *
+         * If you check the file [single-page-application.json] you can see that there's default
+         * value is already populated.
+         */
+        if (!isEmpty(protocolFormValues[ "inboundProtocolConfiguration" ][ "oidc" ][ "callbackURLs" ].filter(x => x))) {
```

Comment:
Let's use `get` from lodash to avoid 
```suggestion
        if (!isEmpty(get(protocolFormValues, "inboundProtocolConfiguration.oidc.callbackURLs")))
```

Reply:
Thanks! This has been resolved in commit 68adef4. I also added some comments to it.